tests/data/plugins/test-plugin/uninstall.php

```diff
@@ -5,7 +5,7 @@
  */
 
 // No direct call.
-if( !defined( 'YOURLS_UNINSTALL_PLUGIN' ) ) die();
+if( !defined( 'YOURLS_UNINSTALL_PLUGIN' ) ) return;
```

Comment:
Not happy about this change, but I don't see how it can pass the unit tests with an `die()` that kills the test process.